services/src/tests/integration/registry/create-resources.spec.ts

```diff
@@ -17,7 +17,7 @@ import {
   UpstreamClientCredentials,
 } from '../../../modules/resource-repository/types';
 import { tmpPoliciesDir } from '../../../modules/config';
-import mockFsForOpa from '../../helpers/mockFsForOpa';
+import * as mockFsForOpa from '../../helpers/mock-fs-for-opa';
```

Comment:
Same as before here, why remove the usage of the default export?

Reply:
The same comment as below :-)